deploy/service/supavisor.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Supavisor
+After=network.target
+Requires=network.target
+StartLimitIntervalSec=5
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=5
+User=root
```

Comment:
I never had to create a service file like this before, but I remember PostgreSQL runs with a different user - the `postgres` user, like the docs suggests: https://www.postgresql.org/docs/current/server-start.html -, and I think that's a good practice.

I wonder if you could run with a non-privilege user here. Do you need something that requires `root`?

Reply:
If I'm not mistaken, only root can listen to ports lower than 1024. I'm considering including a 'stub user' as an example, since we have the option to choose which port Phoenix starts on